src/components/ec-dropdown/__snapshots__/ec-dropdown.spec.js.snap

```diff
@@ -3,7 +3,19 @@
 exports[`EcDropdown #slots should forward CTA slot 1`] = `
 <li
   class="ec-dropdown-search__cta-area"
-  data-test="ec-dropdown-search__cta-area"
```

Comment:
I'm not sure about this addition on the test that was before, can someone confirm it with me?